nomad/state/paginator_test.go

```diff
@@ -7,12 +7,20 @@ import (
 	"github.com/stretchr/testify/require"
 
 	memdb "github.com/hashicorp/go-memdb"
+	"github.com/hashicorp/nomad/helper/uuid"
 	"github.com/hashicorp/nomad/nomad/structs"
 )
 
 func TestPaginator(t *testing.T) {
 	t.Parallel()
-	ids := []string{"0", "1", "2", "3", "4", "5", "6", "7", "8", "9"}
+
+	t.Log("IDs:")
+	ids := []string{}
+	for i := 0; i < 10; i++ {
+		id := uuid.Generate()
+		t.Logf("  %v", id)
+		ids = append(ids, id)
+	}
```

Comment:
I don't know if there's a better way to handle this, but I switched to random UUIDs to avoid relying on any specific input data shape or format. 

Since it's kind of hard to debug test failures with random data I used `t.Log` to print the generated IDs. These are only printed in verbose mode of if a test fails.